ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```diff
@@ -468,6 +468,9 @@ ip prefix-list PL-STATIC-VRF-DEFAULT
 !
 ip route 172.16.0.0/16 172.16.5.4
 ip route 66.66.66.0/24 172.17.0.0
+ip route 10.37.121.1/32 Ethernet3 name IE-ZSCALER
```

Comment:
@gmuloc Should we name these `IE-ZSCALER-PRI/-SEC/-TER` ?

Reply:
Also I think we need to insert the next-hop from `peer_ip`, which is then required for Zscaler IE.

Reply:
`IE-ZSCALER-PRI/-SEC/-TER ` -> I tried but it clashes depending on the order